.gitignore

```diff
@@ -1,5 +1,6 @@
 node_modules/
 npm-debug.log
+yarn-error.log
```

Comment:
Maybe we should just change this to `*.log`?

Reply:
I don't mind calling out these files explicitly, but I'm not fussed. I'll change to `*.log`, we can always change it in the future.